data/hai/hai outfits.txt

```diff
@@ -22,15 +21,14 @@ outfit "Phantom Pallet"
 	"heat generation" 0.03334
 
 	# Replacing 30% of your cargo bay with these triples your cargo scan time.
-	# Do not change these numbers without recalculating the resulting scan time.
 	"cargo space" -1
 	"cargo scan opacity" 15
 
-	"illegal" 12000
-	description "These illegal devices confuse scanners and cargo inspectors seeking contraband. They use holographic visuals, textured force fields, and audio to create realistic-seeming cargo. Hide them in walls, floors, crates, or anywhere else they fit. The projector will generate crates, barrels, piles, and whole shelving units. Beware the fines: the pallets themselves are illegal. Many a captain has lost more money to pallet fines than they've gained from smuggling."
+	description "This Unfettered modification of the standard Hai Holopad is a borderline-illegal smuggling tool."
+	description "Holopads are small versions of the Hai Holovid Zone. They have holographic projectors, textured forcefields, and phase array speakers. As a mid-priced but powerful option, it's often seen in meeting rooms, arcades, and personal gaming setups. Several can be packed together to make a larger holographic space."
+	description "Unfettered Phantom Pallets are more durable and stealthy Holopads. They can hide under floors, in walls, or in ceilings, and remain undetected. Install them in your cargo bay and project fake cargo as sensor ghosts. They confuse scanners into scanning empty space instead of contraband. That can make the difference between seized cargo and delivering needed goods."
```

Comment:
> This Unfettered modification of the standard Hai Holopad is a borderline-illegal smuggling tool.

> Unfettered Phantom Pallets are more durable and stealthy Holopads.

This feels a bit redundant.

Reply:
I rewrote the first and third paragraphs. How are they now?